index.js

```diff
@@ -73,6 +73,6 @@ exports.finish = stats => {
 		`# tests ${passed + failed + skipped}`,
 		`# pass ${passed}`,
 		skipped > 0 ? `# skip ${skipped}` : null,
-		`# fail ${failed + crashed + todo}\n`
-	].filter(Boolean).join('\n');
+		`# fail ${failed + crashed + todo}${os.EOL}`
+	].filter(Boolean).join(os.EOL);
```

Comment:
What does the TAP spec say about line endings? Any chance it mandates `\n`?

Reply:
Haven't seen anything related in docs.